File: user-service/src/main/java/rs/edu/raf/dto/CompanyCreateDTO.java
package rs.edu.raf.dto;

import jakarta.validation.constraints.NotNull;

public record CompanyCreateDTO(
        @NotNull String name,
        @NotNull String phoneNumber,
        @NotNull String address,
        @NotNull String faxNumber,
        @NotNull Integer TIN,
        @NotNull Integer registrationNumber,
        @NotNull Integer businessActivityCode,
        @NotNull Integer registryNumber
) {
}

File: user-service-kotlin/src/main/kotlin/org/example/userservicekotlin/dto/CompanyCreateDTO.kt
package org.example.userservicekotlin.dto

import jakarta.validation.constraints.NotNull

data class CompanyCreateDTO(
    @NotNull val name: String,
    @NotNull val phoneNumber: String,
    @NotNull val address: String,
    @NotNull val faxNumber: String,
    @NotNull val TIN: Int,
    @NotNull val registrationNumber: Int,
    @NotNull val businessActivityCode: Int,
    @NotNull val registryNumber: Int
)
